disnake/message.py

```diff
@@ -896,6 +896,11 @@ class Message(Hashable):
 
     guild: Optional[:class:`Guild`]
         The guild that the message belongs to, if applicable.
+
+    poll: Optional[:class:`Poll`]
+        The poll contained in this message.
+
+        .. versionadded:: 2.10
```

Comment:
Since attribute docs of this sort unfortunately aren't being inherited, this field is missing from `disnake.interactions.base.InteractionMessage`